apiserver/src/config/token.ts

```diff
@@ -0,0 +1,6 @@
+let config = {
+	API_TOKEN_NAME: "test",
```

Comment:
Could you explain to me what does API_TOKEN_NAME refers to ? (I didn't see it used anywhere else than in the index.ts file

Reply:
Useless for the moment, will maybe be used for token renew, if not will be removed too